packages/intent-filler/src/config/chains.toml

```diff
@@ -0,0 +1,44 @@
+[chains.bsc-chapel]
+type = "evm"
+chainId = 97
+stateMachineId = "EVM-97"
+wrappedNativeDecimals = 18
+rpcUrl = "BSC_CHAPEL"
```

Comment:
This should just have the raw rpc URL, so we don't need to extract it from an env file.

Reply:
If this has the raw rpc url, then this file will go into .gitignore?

Reply:
In that case, let's support both env file and raw rpc URL